src/forms/FormFieldCheckbox/FormFieldCheckbox.stories.tsx

```diff
@@ -117,7 +116,7 @@ const kitchenSinkFields = [
 		helperText: 'Helper Text',
 		instructionText: 'InstructionText',
 	}, */
-] as FieldDef<FormFieldCheckboxDef>[];
+] as FieldDef[];

```

Comment:
For consistency please move the "as FieldDef[]"s next to their respective "const fields"